index.html

```diff
@@ -51,9 +51,220 @@ <h2 class="hero-msg">Glad to see you!</h2>
+            </li>
+            <li>
+              <img src="./img/github.svg" alt="github logo" />
+            </li>
+            <li>
+              <img src="./img/angellist.svg" alt="angellist logo" />
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+    <section>
+      <div class="card-container">
+        <div class="card-wrapper">
+          <div class="card">
+            <div class="card-image">
+              <img src="./img/Portfolio.svg" alt="portfolio" />
+            </div>
+            <div class="card-text-container">
+              <div class="card-text">
+                <h6>TONIC</h6>
+              </div>
+              <div class="card-tagline">
+                <div class="card-canopy">
+                  <h6>CANOPY</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>Back End Dev</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>2015</h6>
+                </div>
+              </div>
+              <div class="card-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </div>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="see-link">
+                <a href="#">See Project</a>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="card-container">
+        <div class="card-wrapper">
+          <div class="card">
+            <div class="card-image">
+              <img src="./img/Portfolio2.svg" alt="portfolio" />
+            </div>
+            <div class="card-text-container">
+              <div class="card-text">
+                <h6>Multi-Post Stories</h6>
+              </div>
+              <div class="card-tagline">
+                <div class="card-canopy">
+                  <h6>CANOPY</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>Back End Dev</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>2015</h6>
+                </div>
+              </div>
+              <div class="card-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </div>
```

Comment:
- [ ] Kindly make reference to my comments on line `104` on using a `p` tag here instead of a `div` 👍